src/extension.ts

```diff
@@ -447,8 +450,7 @@ function onConfigurationChange() {
 
     if (serversChanged && bindingChanged) {
       oldConfig = newConfig;
-      updateServerStorage();
-      updateProjectBinding();
+      updateServerStorage().then(() => updateProjectBinding());
```

Comment:
Same here you should be able to do : `updateServerStorage().then(updateProjectBinding);`

But here since you are not doing anything with the result of the promises, if they can be run in parallel, you could do: 
```ts
updateServerStorage();
updateProjectBinding();
```